frontend/src/components/Navbar.js

```diff
@@ -1,18 +1,40 @@
 import React from "react";
-import { connect } from "react-redux";
+import "../style/App.css";
 import { Link } from "react-router-dom";
-import { Navbar, Nav } from "react-bootstrap";
+import { Navbar, Nav, Form, FormControl, Button } from "react-bootstrap";
 
 const LocalNavbar = (props) => {
   return (
-    <Navbar expand="true" bg="dark" variant="dark">
-      <Navbar.Toggle />
+    <Navbar expand="lg" bg="dark" variant="dark" className="container-fluid">
+      <Navbar.Brand href="/home" className="h-hub-brand" title="Home">Housing Hub</Navbar.Brand>
+      <Navbar.Toggle aria-controls="responsive-navbar-nav" />
+      <Navbar.Collapse id="responsive-navbar-nav">
+        <Nav className="flex-lg-fill">
+          <Nav.Link>
+            <Link to="/property/search" className="h-hub-nav-link" title="Search property database">Search</Link>
+          </Nav.Link>
+          <Nav.Link>
+            <Link to="/property/manage" className="h-hub-nav-link" title="Manage properties">My Houses</Link>
+          </Nav.Link>
+          <Nav.Link>
+            <Link to="/about" className="h-hub-nav-link" title="Learn more about Code-For-Charlottesville!">About</Link>
+          </Nav.Link>
+          <Nav.Link>
+            <Link to="/logout" className="h-hub-nav-link">Logout</Link>
+          </Nav.Link> 
+        </Nav> 
+        <Form className="flex-fill justify-content-lg-end" inline>
+          <FormControl type="text" placeholder="Search" className="col-10" />
+          <Button variant="light">
+          <svg className="bi bi-search" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+            <path fill-rule="evenodd" d="M10.442 10.442a1 1 0 0 1 1.415 0l3.85 3.85a1 1 0 0 1-1.414 1.415l-3.85-3.85a1 1 0 0 1 0-1.415z"/>
```

Comment:
Whats going on with this magic?  Can I see a screenshot in the PR?

Reply:
I'm assuming you are referring to the 'svg'. This is Bootstrap's opensource search icon:

https://icons.getbootstrap.com/icons/search/

Admittedly, it is a bit verbose and may be worth just switching out once we pull in another icon library down the line.

Reply:
I might not inline the svg. Create an images directory or add the svgs to that directory if it already exists.